criu/crtools.c

```diff
@@ -255,6 +255,7 @@ int main(int argc, char *argv[], char *envp[])
 			pr_warn("Using -t with criu restore is obsoleted\n");
 
 		ret = cr_restore_tasks();
+		inherit_fd_clean();
```

Comment:
I think we don't need inherit_fd_clean, but if you want to have it, it will be better to call it from cr_restore_tasks.

Reply:
Removed it altogether.